### DISCUSSION TITLE
How to edit functionality of  clone button in content item in orchadcore cms.

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#11847] [Q&A]
Hi I want to edit the functionality of clone button of content item in orchadcore cms so that I can add a popup when clone button is clicked can you please let me know which part of the code to edit for it ?

**Comment by hishamco** [comment]
You can check ContentsButtonActions_SummaryAdmin.cshtml in `OrchardCore.Contents` module

**Reply by syedshadab19** [reply]
ok will check it
